### ISSUE TITLE
layer normalization

### ISSUE THREAD

**Issue by suiyuan2009** [tensorflow/tensorflow#3621]
recently Hinton published layer normalization on [arxiv layer normalization](https://arxiv.org/pdf/1607.06450v1.pdf?), any plan to add an Op?

**Comment by suiyuan2009** [comment]
I see c++ implement of batch_normalization is deprecated

**Comment by suiyuan2009** [comment]
@xodus7 great

**Comment by xcyan** [comment]
@suiyuan2009 Have you ever tried this layer_norm op with most recent TF (1.3 or 1.4)? 
#3671 #14562
